### ISSUE TITLE
Timeouts occur after installing too many packages

### ISSUE THREAD

**Issue by HeyZoos** [GlitchEnzo/NuGetForUnity#465]
## Description
After installing enough packages, the `GetUpdates` request times out causing all actions to fail.

### My packages

```
<?xml version="1.0" encoding="utf-8"?>
<packages>
  <package id="Amazon.Extensions.CognitoAuthentication" version="2.2.3" />
  <package id="AWSSDK.CognitoIdentity" version="3.7.0.203" />
  <package id="AWSSDK.CognitoIdentityProvider" version="3.7.0.2" />
  <package id="AWSSDK.Core" version="3.7.13.1" />
  <package id="AWSSDK.SecurityToken" version="3.7.1.196" />
  <package id="LanguageExt.Core" version="4.2.9" />
  <package id="Microsoft.Bcl.AsyncInterfaces" version="5.0.0" />
</packages>
```

### Error
```
Unable to retrieve package list from http://www.nuget.org/api/v2/GetUpdates()?packageIds='Amazon.Extensions.CognitoAuthentication|AWSSDK.CognitoIdentity|AWSSDK.CognitoIdentityProvider|AWSSDK.Core|AWSSDK.SecurityToken|LanguageExt.Core|Microsoft.Bcl.AsyncInterfaces'&versions='2.2.3|3.7.0.203|3.7.0.2|3.7.13.1|3.7.1.196|4.2.9|5.0.0'&includePrerelease=false&includeAllVersions=false&targetFrameworks=''&versionConstraints=''
System.Net.WebException: The operation has timed out.
  at System.Net.HttpWebRequest.RunWithTimeoutWorker[T] (System.Threading.Tasks.Task`1[TResult] workerTask, System.Int32 timeout, System.Action abort, System.Func`1[TResult] aborted, System.Threading.CancellationTokenSource cts) [0x000f8] in <a6f433c7914f4a8aa11562de443cf6c1>:0 
  at System.Net.HttpWebRequest.GetResponse () [0x00016] in <a6f433c7914f4a8aa11562de443cf6c1>:0 
  at NugetForUnity.NugetHelper.RequestUrl (System.String url, System.String userName, System.String password, System.Nullable`1[T] timeOut) [0x000c0] in <52f11524b7d74432afd0486e05bd92cc>:0 
  at NugetForUnity.NugetPackageSource.GetPackagesFromUrl (System.String url, System.String username, System.String password) [0x0006c] in <52f11524b7d74432afd0486e05bd92cc>:0 
  at NugetForUnity.NugetPackageSource.GetUpdates (System.Collections.Generic.IEnumerable`1[T] installedPackages, System.Boolean includePrerelease, System.Boolean includeAllVersions, System.String targetFrameworks, System.String versionContraints) [0x00112] in <52f11524b7d74432afd0486e05bd92cc>:0 
UnityEngine.Debug:LogErrorFormat (string,object[])
NugetForUnity.NugetPackageSource:GetUpdates (System.Collections.Generic.IEnumerable`1<NugetForUnity.NugetPackage>,bool,bool,string,string)
NugetForUnity.NugetHelper:GetUpdates (System.Collections.Generic.IEnumerable`1<NugetForUnity.NugetPackage>,bool,bool,string,string)
NugetForUnity.NugetWindow:UpdateUpdatePackages ()
NugetForUnity.NugetWindow:Refresh (bool)
NugetForUnity.NugetWindow:OnEnable ()
UnityEditor.EditorWindow:GetWindow<NugetForUnity.NugetWindow> ()
NugetForUnity.NugetWindow:DisplayNugetWindow ()
```

- **NuGetForUnity Version:** 3.0.5
- **Unity Version:** 2021.3.8f1
- **Operating System:** Windows

**Comment by JoC0de** [comment]
Hi @HeyZoos thanks for the Issue and sorry for the late response. In #498 I increased the timeout to 10 seconds, before it was only 5 seconds.

**Comment by JoC0de** [comment]
fixed in 3.1.0
